pkg/vere/main.c

```diff
@@ -367,9 +367,9 @@ _main_getopt(c3_i argc, c3_c** argv)
       }
       case c3__loom: {
         c3_w lom_w;
-        c3_o res_o = _main_readw(optarg, u3a_bits + 3, &lom_w);
+        c3_o res_o = _main_readw(optarg, u3a_bits_max+1, &lom_w);
         if ( (c3n == res_o) || (lom_w < 20) ) {
-          fprintf(stderr, "error: --loom must be >= 20 and <= %u\r\n", u3a_bits + 2);
+          fprintf(stderr, "error: --loom must be >= 20 and <= %u\r\n", u3a_bits_max);
```

Comment:
This change needs to also be applied to all the other subcommands that use `--loom` as an argument (`_cw_meld()`, `_cw_pack()`, &c).

Reply:
done.